### ISSUE TITLE
Request for Parallel Processing in Web Service

### ISSUE THREAD

**Issue by RishishPandey** [ahmetoner/whisper-asr-webservice#80]
Hi,

I recently used your web service and attempted to make multiple requests together. However, I noticed that the service handled them in a sequential manner only. I was wondering if it's possible to process them in parallel instead?

As a newcomer to this field, I'm not sure if multiple GPUs are necessary to achieve parallel processing. If that's the case, please let me know.

Thank you for your time and assistance. Please feel free to ask if you need any further information.

Best regards,
Rishish Pandey

**Comment by ostrolucky** [comment]
Indeed. When container processes a request, even calling /docs endpoint fails
